src/test/kotlin/org/rust/completion/RustDeriveCompletionTest.kt

```diff
@@ -0,0 +1,43 @@
+package org.rust.completion
+
+import com.intellij.codeInsight.completion.CompletionType
+import org.rust.lang.RustTestCaseBase
+
+class RustDeriveCompletionTest : RustTestCaseBase() {
```

Comment:
You should be able to inherit from `RustCompletionTestBase` and just write

```
fun testCompleteOnStruct = checkSoleCompletion()
```

Or is this not working because there several variants of completion? In any case it's better to remove this duplication in tests functions. See `checkSoloCompletion` for how it can be done (basically, you can get the name of the test file from the name of the test function).